en/guides/user-management/README.md

```diff
@@ -0,0 +1,127 @@
+[managing](/en/course-management#managing-course-users) of 
+[course users](/en/course-management#course-user) and [appointing](/en/course-management#appointing-course-managers) course managers is discussed in [course management](/en/course-management).
+:::
+
+## Navigating to a user
+
+As a [teacher](#teacher), you navigate to the [user overviewt](#user-overview) by selecting <span class="guilabel">Users</span> in the [management menu](#management-menu) at the right side of the [navigation panel](/en/for-students#navigation-panel).
+
+![admin menu users](./staff.admin_menu_users.en.png)
+
+## User overview
+In the **user overview**, the user's username, real name, email and [progress statistics](#user-progress-statistics) are listed from the [user profile](/en/for-students#user-profile) from all users. In the first column, you can find an [icon](#user-permissions-icon) that corresponds to the [management permissions](#management-permissions) that have been [assigned](#assign-permissions) to that user.
+
+![image](./staff.users.en.png)
+
+## Searching a user
+Use the search bar at the top of the [user overview](#user-overview) to search for specific users based on their usernames, names or email addresses.
+
+![image](./staff.users_filtered.en.png)
+
+## Selecting a user
+Click on the username to [navigate](#navigating-to-a-user) to the [profile page](/en/for-students#profile-page) of te user.
+
+![image](./staff.users_filtered_link.en.png)
+
+## User progress statistics voortgangsstatistieken
+The **progress statistics** of a user consists of two numbers $c/s$, where $s$ is the number of [exercises](/en/for-students#exercise) for which that user has [submitted](/en/for-students#submitting-a-solution) atleast one [solution](/en/for-students#solution) and $c$ is the number of [exercises](/en/for-students#exercise) for which that user has [submitted](/en/for-students#submitting-a-solution) atleast one *correct* [solution](/en/for-students#solution).
+
+
+## Editing a user profile
+As a [teacher](#teacher), you can edit the [user profile](/en/for-students#user-profile) of [students](#student) and other teachers. To do this, click the edit button in the top right corner of the [profile page](/en/for-students#profile-page) of the user.
+
+![image](./staff.user_edit_link.en.png)
+
+Alternatively, you can press the edit button at the right side of a user in the [user overview](#user-overview) to immediately edit their [user profile](/en/for-students#user-profile)
+
+![image](./staff.users_filtered_edit_link.en.png)
+
+## Management permissions
+
+In order to perform certain action on Dodona, you require additional **management permissions**. As a [teacher](#teacher), you can use the property <span class="guilabel">Permissions</span> on the [profile page](/en/for-students#profile-page) of a user to find out which permissions that user has.
+
+![image](./staff.user_edit_permission.en.png)
+
+## User permissions icon
```

Comment:
The icons have changed too. There are now two options: course admins (with icon) and students (without icon).
![tYKp](https://user-images.githubusercontent.com/3226995/90105855-dfc68500-dd46-11ea-877c-9d4f60a55cfa.png)

Reply:
``` ruby
          <% if user.zeus? %>
            <i class="mdi mdi-android mdi-18" title='<%= t ".zeus" %>'></i>
          <% end %>,
```
 een cursusbeheerder gaat nooit dat icoontje zien?

Reply:
No, the code responsible for the icons is here:
https://github.com/dodona-edu/dodona/blob/e25509f4862949d01f10c45e1b51ba896c326e64/app/views/course_members/_members_table.html.erb#L19-L24

Only course admins and users not connected to an institution will have an icon. But the latter doesn't need to be documented.